h2o-hadoop-common/tests/python/pyunit_gbm_on_hive.py

```diff
@@ -30,24 +30,19 @@ def gbm_on_hive():
 
     # read from S3
     airlines_dataset_original = h2o.import_file(path="https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/AirlinesTest.csv.zip")
-    # read from Hive Distributed
```

Comment:
why is this deleted? I think it should stay

Reply:
moved to a separate test